README.rst

```diff
@@ -30,11 +30,18 @@ systems have packages yet, we provide a temporary solution via the
 ``letsencrypt-auto`` wrapper script, which obtains some dependencies
 from your OS and puts others in a python virtual environment::
 
-  user@webserver:~$ git clone https://github.com/letsencrypt/letsencrypt
-  user@webserver:~$ cd letsencrypt
-  user@webserver:~/letsencrypt$ ./letsencrypt-auto --help
+  user@webserver:~$ wget https://dl.eff.org/certbot-auto
+  user@webserver:~$ chmod a+x ./certbot-auto
+  user@webserver:~$ ./certbot-auto --help
 
-Or for full command line help, type::
+.. hint:: The certbot-auto download is protected by HTTPS, which is pretty good, but if you'd like to
+          double check the integrity of the ``certbot-auto`` script, you can use these steps for verification before running it::
+
+            user@server:~$ wget https://dl.eff.org/certbot-auto.sig
+            user@server:~$ gpg2 --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
+            user@server:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.sig certbot-auto
```

Comment:
You're right, gpg and openssl/smime have different signature formats. We could keep the openssl one the releases currently use, and that's probably the best idea, but it'll prevent people from using the PGP web of trust as a second verification method.  It's probably good enough to give them a URL on some independent domain (letsencrypt.org, perhaps) to fetch the openssl key from.